### PR TITLE
Translation of featured_chapters to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586

Please review.

### CODE REVIEW

File: src/templates/ja/2019/featured_chapters.html [hunk 1]
```diff
@@ -10,72 +10,72 @@
...
 {%- elif featured_chapter == "css" %}
   {%- set featured_chapter_name = "CSS" %}
-  {%- set featured_chapter_quote = "Cascading Style Sheets (CSS) are used to paint, format, and layout web pages. Their capabilities span concepts as simple as text color to 3D perspective. It also has hooks to empower developers to handle varying screen sizes, viewing contexts, and printing. CSS helps developers wrangle content and ensure it's adapting properly to the user." %}
-  {%- set featured_chapter_stats = {"stat1":"5%","label1":"Pages using custom properties","stat2":"2%","label2":"Sites that use CSS Grid","stat3":"780","label3":"Number of digits in largest Z-Index value"} %}
+  {%- set featured_chapter_quote = "Cascading Style Sheets (CSS) は、Web ページのペイント、フォーマット、レイアウトに使用されます。その機能は、テキストの色から3Dパースペクティブと、シンプルな概念にまで及びます。また、さまざまな画面サイズ、表示コンテキスト、および印刷を処理するために開発者に力を与えるフックを持っています。CSS は、開発者がコンテンツを調整し、ユーザーに適切に適応することを確認するのに役立ちます。" %}
+  {%- set featured_chapter_stats = {"統計1":"5%","ラベル1":"Pages using custom properties","統計2":"2%","ラベル2":CSS Gridを使用しているサイト"","統計3":"780","ラベル3":"Z-Indexの最大値の桁数"} %}
```
Comment (bazzadp) [inline src/templates/ja/2019/featured_chapters.html:14]: The statX and labelX are all variables so need to leave them as they are originally. Can you change them all back for all lines?

Also this CSS line has a Quote issue with label2 - fixed below. The rest don't have this issue and look good.

```suggestion
  {%- set featured_chapter_stats = {"stat1":"5%","label1":"Pages using custom properties","stat2":"2%","label2":"CSS Gridを使用しているサイト","stat3":"780","label3":"Z-Indexの最大値の桁数"} %}
```

File: src/templates/ja/2019/featured_chapters.html [hunk 2]
```diff
@@ -10,72 +10,72 @@
...
 {%- elif featured_chapter == "performance" %}
-  {%- set featured_chapter_name = "Performance" %}
-  {%- set featured_chapter_quote = "Performance is a visceral part of the user experience. For many websites, an improvement to the user experience by speeding up the page load time aligns with an improvement to conversion rates. Conversely, when performance is poor, users don't convert as often and have even been observed to be rage clicking on the page in frustration." %}
-  {%- set featured_chapter_stats = {"stat1":"13%","label1":"Sites with fast FCP","stat2":"42%","label2":"Sites with slow TTFB","stat3":"40%","label3":"Sites with fast FID"} %}
+  {%- set featured_chapter_name = "パフォーマンス" %}
+  {%- set featured_chapter_quote = "パフォーマンスはユーザー体験の内臓的な部分です。多くのウェブサイトでは、ページの読み込み時間を高速化してユーザー体験を向上させることは、コンバージョン率の向上につながります。逆に、パフォーマンスが悪いと、ユーザーは頻繁にコンバージョンしないし、不満でページを激怒してクリックしてしまうことさえ観察されています。" %}
+  {%- set featured_chapter_stats = {"統計1":"13%","ラベル1":"Sites with fast FCP","統計2":"42%","ラベル2":"Sites with slow TTFB","統計3":"40%","ラベル3":"FIDが速いサイト"} %}
```
Comment (bazzadp) [inline src/templates/ja/2019/featured_chapters.html:34]: Missed translating the first two labels.

File: src/templates/ja/2019/featured_chapters.html (hunk 1 above)
Comment (ksakae1216) [reply src/templates/ja/2019/featured_chapters.html:14]: Thanks, I fix.

File: src/templates/ja/2019/featured_chapters.html (hunk 2 above)
Comment (ksakae1216) [reply src/templates/ja/2019/featured_chapters.html:34]: Thanks, I translated.

**Pull request merged**